### ISSUE TITLE
A rule of min=0 is not applied to the element's rule set

### ISSUE THREAD

**Issue by tonineumann** [jquery-validation/jquery-validation#897]
In v1.11.1 of the plug in. Setting an attribute rule of min="0"  results in the rule being ignored and not being added to the rule set.

``` html
<input class="number" type="text" min="0" max="100" />
```

At line 868 the if test:

``` js
if ( value ) {
...
```

Since the value is 0, the if test evaluates to false.  I believe the solution is to test value against undefined.

``` js
if ( value !== undefined ) {
...
```

Please see this jsfiddle http://jsfiddle.net/WKnaE/ which reproduces this issue.

Expected results:
1) Enter a number greater than 100.  Tab out of the input. Results in an error message.
2) Enter a number less than 0. Tab out of the input.  Results in no error.  The validation should cause an error to be displayed, but it is not.
